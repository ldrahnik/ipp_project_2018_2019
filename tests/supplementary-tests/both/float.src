.IPPcode19
DEFVAR GF@float
MOVE GF@float float@0x1.2000000000000p+0
WRITE GF@float
READ GF@float float
DEFVAR GF@int
FLOAT2INT GF@int GF@float
WRITE GF@int
INT2FLOAT GF@float GF@int
WRITE GF@float
