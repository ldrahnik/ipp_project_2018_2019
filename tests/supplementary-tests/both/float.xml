<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
 <instruction order="1" opcode="DEFVAR">
  <arg1 type="var">GF@float</arg1>
 </instruction>
 <instruction order="2" opcode="MOVE">
  <arg1 type="var">GF@float</arg1>
  <arg2 type="float">0x1.2000000000000p+0</arg2>
 </instruction>
 <instruction order="3" opcode="WRITE">
  <arg1 type="var">GF@float</arg1>
 </instruction>
 <instruction order="4" opcode="READ">
  <arg1 type="var">GF@float</arg1>
  <arg2 type="type">float</arg2>
 </instruction>
 <instruction order="5" opcode="DEFVAR">
  <arg1 type="var">GF@int</arg1>
 </instruction>
 <instruction order="6" opcode="FLOAT2INT">
  <arg1 type="var">GF@int</arg1>
  <arg2 type="var">GF@float</arg2>
 </instruction>
 <instruction order="7" opcode="WRITE">
  <arg1 type="var">GF@int</arg1>
 </instruction>
 <instruction order="8" opcode="INT2FLOAT">
  <arg1 type="var">GF@float</arg1>
  <arg2 type="var">GF@int</arg2>
 </instruction>
 <instruction order="9" opcode="WRITE">
  <arg1 type="var">GF@float</arg1>
 </instruction>
</program>
