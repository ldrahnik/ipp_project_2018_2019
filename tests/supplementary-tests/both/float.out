1.12511.0